rbac/management/management/commands/migrate_relations.py

```diff
@@ -15,8 +17,13 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         """Add arguments to command."""
         parser.add_argument("--org-list", nargs="+", default=[])
-        parser.add_argument("--exclude-apps", nargs="+", default=[])
-        parser.add_argument("--write-to-db", default=False, action="store_true")
+        parser.add_argument(
+            "--exclude-apps",
+            nargs="+",
+            default=settings.V2_MIGRATION_APP_EXCLUDE_LIST,
```

Comment:
Should we add this in the internal/Turnpike endpoint defaults as well? https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/internal/views.py#L490